src/index.js

```diff
@@ -219,13 +221,16 @@ f.pinMode = function (pin, direction, mux, pullup, slew, callback) {
                 } else
                     callback(resp.err, resp.value);
             }
-            return (resp.value);
+            // Move on for now.
+            // return (resp.value);
```

Comment:
I don't think this is necessary anymore.

Reply:
Fixed